datafusion/row/src/lib.rs

```diff
@@ -221,6 +221,56 @@ mod tests {
         vec![Some(5), Some(7), None, Some(0), Some(111)]
     );
 
+    #[test]
+    #[allow(non_snake_case)]
```

Comment:
Why is this annotation needed?

```
    #[allow(non_snake_case)]
```


I removed it locally and clippy till passes